brille_spinwaves/run_matlab_test.ps1

```diff
@@ -0,0 +1,37 @@
+. $PSScriptRoot/../powershell_scripts/powershell_helpers.ps1 <# Imports:
```

Comment:
I know this isn't my PR to review, but I noticed that in the Jenkinsfile the `execute_matlab_command.ps1` script is being used from `PACE-jenkins-shared-library`, so I think this file can be removed!